src/content/peerconnection/constraints/js/main.js

```diff
@@ -290,10 +290,9 @@ setInterval(function() {
 // might be named toString?
 function dumpStats(results) {
   var statsString = '';
-  Object.keys(results).forEach(function(key, index) {
-    var res = results[key];
+  results.forEach(function(res) {
     statsString += '<h3>Report ';
-    statsString += index;
+    statsString += res.type;
```

Comment:
changed the output format slightly in 486ee457d30f374247a77f37b4cf6234cf7dde55 -- now type, timestamp and id are special